src/dbSta/test/report_cell_usage_modinsts.ok

```diff
@@ -0,0 +1,25 @@
+[INFO ODB-0227] LEF file: liberty1.lef, created 2 layers, 6 library cells
+Cell type report:                       Count       Area
+  Buffer                                    4    4000.00
+  Sequential cell                           2    2000.00
+  Total                                     6    6000.00
+
+Cell instance report:
+  snl_bufx1                                 4    4000.00
+  snl_ffqx1                                 2    2000.00
+Cell type report:                       Count       Area
```

Comment:
If the report is not for the top module I think it would be good to indicate that in the report.  Its confusing to see these three reports and only by looking at the tcl can you see what they mean.